ethcore/vm/src/action_params.rs

```diff
@@ -112,6 +112,14 @@ impl Default for ActionParams {
 	}
 }
 
+impl ActionParams {
+	pub fn from_origin(mut self, address: Address) -> Self {
```

Comment:
Seems to be used only for tests, might be misleading to include it everwhere in the code.